camp/ocampbell/xBoxTriggerProject/src/main/java/frc/robot/commands/ControlledFalconDefault.java

```diff
@@ -0,0 +1,88 @@
+// ************************************************************
+// Bishop Blanchet Robotics
+// Home of the Cybears
+// File: ControlledFalconDefault.java
+// Intent: Assigns a motor speed value to a falcon.
+// ************************************************************
+
+// ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ 
+package frc.robot.commands;
+
+import frc.robot.control.SubsystemCollection;
+import frc.robot.subsystems.ControlledFalcon;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class ControlledFalconDefault extends CommandBase {
+
+  private SubsystemCollection subsystems;
+
+
+  private ControlledFalcon motor;
+  private double speed = 0.0;
+  private boolean done = false;
+  private double lTrigger = 0.0;
+  private double rTrigger = 0.0;
+
+
+  public ControlledFalconDefault(SubsystemCollection collection, ControlledFalcon motor, double lTrigger, double rTrigger) {
+      this.motor = motor;
+      this.lTrigger = lTrigger;
+      this.rTrigger = rTrigger;
+
+    addRequirements(motor);
+    subsystems = collection;
+}
+
+  @Override
+  public void initialize() {}
+
+
+  @Override
+  public void execute() {
+
+    lTrigger = -lTrigger;
+
+    //delta approach
+    speed = lTrigger + rTrigger;
```

Comment:
It is my understanding that triggers on xbox controller will only ever be positive.  What needs to change in this line to respect the requirements specified in the mini project?

![image](https://github.com/Team4682CyBears/Crescendo2024/assets/56608471/83cf24ea-b5ee-446d-9914-cab32387f615)